packages/common-db/package.json

```diff
@@ -32,13 +32,13 @@
     "url": "https://github.com/ioBroker/ioBroker.js-controller/packages/common"
   },
   "scripts": {},
-  "main": "index.js",
+  "main": "build/index.js",
   "license": "MIT",
```

Comment:
TypeScript packages should have a
```
"types": "build/index.d.ts",
```
field so the dependent packages can pick up generated type declarations.